XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,573 @@
+* Creating a `CredentialDelete` transaction type
+
+This feature will require an amendment, tentatively titled `featureCredentialAuth`.
+
+### 1.1. Background: DIDs and Verifiable Credentials (VCs)
+
+A [verifiable credential](https://en.wikipedia.org/wiki/Verifiable_credentials) (VC), as defined by the [W3C specification](https://www.w3.org/TR/vc-data-model-2.0/), is a secure and tamper-evident way to represent information about a subject, such as an individual, organization, or even an IoT device. These credentials are issued by a trusted entity and can be verified by third parties without directly involving the issuer at all.
+
+For a clearer understanding of the relationship between [DIDs (Decentralized Identifiers)](https://en.wikipedia.org/wiki/Decentralized_identifier) and VCs, a real-world example is considered. A DID serves as a unique identifier for an entity, similar to a fingerprint or a photo of a person (their physical characteristics). In contrast, a VC acts as a verifiable piece of information associated with that DID, much like a driver's license or passport. When a third party needs to verify an identity claim, they can examine the VC presented by the entity and ensure it aligns with the corresponding DID (such as comparing the photo on a driver's license to the physical characteristics of the person to whom it supposedly belongs).
+
+### 1.2. Terminology
+
+* **Credential**: A representation of one or more assertions made by an issuer. For example, a passport is a representation of the assertion that a person is a citizen of a given country.
+* **Issuer**: The account that creates (issues) the credential. For example, a passport is issued by a country's government.
+* **Subject**: The account that the credential is issued to. For example, a passport is issued to a specific person.
+* **Whitelist**: A list of accounts that are allowed to do a given action, or (if used as a verb) the action of adding an account to said list.
+* **Preauthorization**: Essentially the same thing as whitelisting.
+
+*Note: These definitions do not directly match the definitions listed in the [W3C Verifiable Credentials spec](https://www.w3.org/TR/vc-data-model-2.0/#terminology). These terms are used in a slightly different way in the context of this spec.* 
+
+### 1.3. Basic Flow
+
+In this scenario, an authorizer/verifier account, Verity, is a regulated entity wants to ensure that she is only interacting with properly KYC'd accounts.
+
+* Verity sets up her account such that she only wants to interact with properly KYC'd accounts. He trusts Isabel, a credential issuer, to properly vet accounts and issue credentials attesting to that.
+* Alice, who wants to interact with Verity's account, submits her KYC documents to Isabel off-chain. Isabel creates a credential attesting to her trustworthiness on-chain.
+* Alice accepts the credential, making it a valid credential.
+* Alice can now interact with/send funds to Verity.
```

Comment:
This section is somewhat confusingly structured and the pronouns for Verity change genders partway through. Also, typo in first sentence: missing word in "is a regulated entity [who/that] wants". I suggest first introducing the parties/roles and then describing the flow. The following is a suggestion of how you could reword it:

```suggestion
An example scenario features three parties with different roles:

* Verity is a regulated business that wants to interact only with properly KYC'd accounts, to ensure legal compliance. This makes Verity an _authorizer_ because they configure which accounts are allowed (authorized) to interact with them.
* Isabel is a credential issuer that vets accounts and issues credentials on-chain attesting that the accounts are who they say they are.
* Alice is a user who wants to interact with Verity.

The authorization flow in this scenario works as follows:

1. Verity sets up their account so that only authorized accounts can interact with it, and configures it to accept credentials issued by Isabel.
2. Alice submits her KYC documents to Isabel privately, off-chain.
3. Isabel examines Alice's documents and creates a credential on-chain attesting to Alice's trustworthiness.
4. Alice accepts the credential, making it valid.
5. Alice can now send funds to Isabel.

Importantly, the documents that Alice sends to Isabel can include personally identifiable or private information that's needed to verify Alice's identity, but this information is never published or stored on chain and Verity never needs to see it. Also, other businesses that trust Isabel can accept the same credentials so Alice does not need to repeatedly re-verify for every party she wants to interact with.
```

I also added a summary of some of the advantages of this flow for context. Hopefully those are accurate.